x-pack/plugins/apm/server/routes/service_map/route.ts

```diff
@@ -146,6 +144,7 @@ const serviceMapRoute = createApmServerRoute({
       start,
       end,
       maxNumberOfServices,
+      serviceGroup,
     });
```

Comment:
Why do we pass both serviceNames and serviceGroup?

Reply:
Service name is used when we are in the context of a service. We could rename it to `serviceName`. Service group is passed to apply a `kuery` filter for the group